r/R/dplyr.R

```diff
@@ -216,6 +215,52 @@ column_select <- function(.data, ..., .FUN = vars_select) {
   .data
 }
 
+relocate.arrow_dplyr_query <- function(.data, ..., .before = NULL, .after = NULL) {
+  .data <- arrow_dplyr_query(.data)
+
+  # TODO: look for unsupported tidyselect selection helpers in exprs(...),
+  # .before, and .after and throw an error if detected
```

Comment:
Should I dig into this now or create a Jira for follow-up later?

Reply:
What is unsupported?

Reply:
See the examples below `# Can also select variables based on their type` at https://dplyr.tidyverse.org/reference/relocate.html#examples. None of those currently work here.

Reply:
Yeah, those won't work. Error should be expected, preferably with a useful message. Followup JIRA sounds right, it's a bigger issue and not confined to relocate()

Reply:
Checks for unsupported tidyselect helpers added in 253f44d51f641079abfba591e4ff7628d685c948. These rely on the utility functions added in ed0ca40c677651409acaccda32ef499fce380edc.